connector/core/data-migration/src/main/java/org/oaebudt/edc/data/migration/AbstractPostgresqlMigrationExtension.java

```diff
@@ -0,0 +1,60 @@
+package org.oaebudt.edc.data.migration;
+
+import org.eclipse.edc.runtime.metamodel.annotation.Extension;
+import org.eclipse.edc.runtime.metamodel.annotation.Inject;
+import org.eclipse.edc.spi.monitor.Monitor;
+import org.eclipse.edc.spi.persistence.EdcPersistenceException;
+import org.eclipse.edc.spi.system.ServiceExtension;
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.sql.DriverManagerConnectionFactory;
+import org.eclipse.edc.sql.datasource.ConnectionFactoryDataSource;
+import org.eclipse.edc.transaction.datasource.spi.DataSourceRegistry;
+
+import java.util.Properties;
+
+import static java.util.Optional.ofNullable;
+import static org.flywaydb.core.api.MigrationVersion.LATEST;
+
+@Extension(AbstractPostgresqlMigrationExtension.NAME)
+public class AbstractPostgresqlMigrationExtension implements ServiceExtension {
```

Comment:
why is this called "abstract"? it's not an abstract class

Reply:
Resolved